app/js/views/add_todo_view.js

```diff
@@ -0,0 +1,41 @@
+define([
+  'jquery',
+  'backbone',
+  'text!templates/app_view.ejs',
+  'models/todo'
+], function ($, Backbone, indexTemplate, ToDo) {
+  return Backbone.View.extend({
+    events: {
+      submit: 'createNew'
+    },
+
+    input: this.$('#todo-new-input'),
```

Comment:
This does not work like you would expect. `this` when the class is evaluated will refer to `window` which is the global object. This can cause errors like: If the element with id `#todo-new-input` does not exist when the class is evaluated, the view will not have an element and will not be visible when rendered.